tests/cdb2sql.test/t03.multiline.sql

```diff
@@ -1,4 +1,2 @@
--- test complete statment spanning multiple lines that contains
--- a non-terminating semicolon
```

Comment:
Needed to take the leading comments out of the test cases because when their contents are provided via argv, cdb2sql reads '-' and thinks that we want to start an interactive session.

I don't think that these comments were very useful because the test cases are pretty simple.